cypress/integration/console/devices/create-repository-registration.spec.js

```diff
@@ -601,7 +532,7 @@ describe.skip('End device repository create', () => {
         cy.findByLabelText('End device ID').type(devId2)
         cy.findByLabelText('View registered end device').check()
 
-        cy.findByRole('button', { name: 'Register end device' }).click()
+        cy.findByRole('button', { name: 'Add end device' }).click()
```

Comment:
I think we should actually try doing this three times. Rish was notifying me about an issue there after the third time.